README.md

```diff
@@ -21,30 +37,119 @@ from fastapi_nextauth_jwt import NextAuthJWT
 app = FastAPI()
 
 JWT = NextAuthJWT(
-    secret="y0uR_SuP3r_s3cr37_$3cr3t",
+    secret="y0uR_SuP3r_s3cr37_$3cr3t",  # Better to use environment variable
 )
 
 @app.get("/")
-async def return_jwt(jwt: Annotated[dict, Depends(JWT)]):
-    return jwt
+async def return_jwt_get(jwt: Annotated[dict, Depends(JWT)]):
+    return {"message": f"Hi {jwt['name']}. Greetings from fastapi!"}
+
+# For CSRF protection testing
+@app.post("/")
+async def return_jwt_post(jwt: Annotated[dict, Depends(JWT)]):
+    return {"message": f"Hi {jwt['name']}. Greetings from fastapi!"}
+```
+
+## Configuration Options
+
+### Essential Settings
+
+- **secret** (str): The secret key used for JWT operations. Should match `NEXTAUTH_SECRET` in your Next.js app.
+  ```python
+  JWT = NextAuthJWT(secret=os.getenv("NEXTAUTH_SECRET"))
+  ```
+
+### Security Options
+
+- **csrf_prevention_enabled** (bool): Toggle CSRF protection
+  - Defaults to `True` in development (`ENV=dev`), `False` otherwise
```

Comment:
Sorry, this is incorrect in the docstring. It should be: 

```suggestion
- **csrf_prevention_enabled** (bool): Enable CSRF protection
  - Defaults to `False` in development (`ENV=dev`), `True` otherwise
```